lib/diagnostic.js

```diff
@@ -8,15 +8,25 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
```

Comment:
Structure your code using whitespace and indentation to increase readability.  While this function is correct, it should be written for readability as well.
```javascript
const min = function min() {
  let result = arguments[0];
  
  for (let i = 1; i < arguments.length; i++) {
    if (arguments[i] < result) {
      result = arguments[i];
    }
  }
  
  return result;
};
```